src/Umbraco.Web.UI.Client/src/less/components/card.less

```diff
@@ -84,63 +85,73 @@
 	padding: 0;
 	margin: 0 auto;
 	list-style: none;
-
+    width: 100%;
+    
 	display: flex;
 	flex-flow: row wrap;
 	justify-content: flex-start;
 }
 
 .umb-card-grid li {
-	padding: 5px;
 	overflow: hidden;
 	font-size: 12px;
 	text-align: center;
-	width: 100px;
 	box-sizing: border-box;
 	position: relative;
+    width: 100px;
 }
 
-.umb-card-grid li.-four-in-row {
+.umb-card-grid.-four-in-row li {
     flex: 0 0 25%;
     max-width: 25%;
 }
 
-.umb-card-grid li.-three-in-row {
+.umb-card-grid.-three-in-row li {
 	flex: 0 0 33.33%;
     max-width:33.33%;
 }
 
 .umb-card-grid .umb-card-grid-item {
+    position: relative;
 	display: block;
 	width: 100%;
-	height: 100%;
+	//height: 100%;
+    padding-top: 100%;
```

Comment:
@nielslyngsoe it seems this `padding-top: 100%` afffect field types in Umbraco Forms (which use the `.umb-card` styles.

![image](https://user-images.githubusercontent.com/2919859/66083575-df792d00-e56c-11e9-980e-51b9afcc0510.png)

Reply:
Hi @bjarnef thanks, The issue is also reported on the Forms Issue Tracker. https://github.com/umbraco/Umbraco.Forms.Issues/issues/217

I hope we will get it fixed soon — We shouldn't be using css-classes directly between projects, only components, but its not always the case.